pkg/api/aim/response/project.go

```diff
@@ -1,5 +1,22 @@
 package response
 
+// ProjectActivity represents the response json for the `GET aim/projects/activity` endpoint.
+type ProjectActivity struct {
```

Comment:
can you please add `Response` to the end of the name to be consistent with `ProjectParamsResponse` and what we have in `mflow` part?

Reply:
I can but I do think it is redundant since the package is already named "response"

Reply:
that's a good point probably and I fully agree with you. I don't remember why I added `Response` to the end, maybe something was misleading but it seems like we can remove it. Right now we have 2 ways:
- can you please add it since we have it everywhere  so everything will be just the same. Then you can create a task to normalise it.
- if you want to remove - can you please remove it from everywhere.

Reply:
added